components/SplitScreen/SplitForm.js

```diff
@@ -321,14 +374,49 @@ export default () => {
         }
         reveal={
           converterStatus.status === CONVERTER_STATUSES.FORM ? null : (
-            <Converter />
+            <Converter
+              amountRequested={amountRecipient}
+              backToSplit={() => null}
+              toAnj={forwards}
+              transactionHash={transactionHash}
+              isFinal={isFinal}
+            />
           )
         }
       />
     </div>
   )
 }
 
+function LabelWithOverlay({ label, description, overlayPlacement }) {
+  return (
+    <OverlayTrigger
+      placement={overlayPlacement}
+      delay={{ hide: 400 }}
+      overlay={props => (
+        <Tooltip {...props} show="true">
+          {description}
+        </Tooltip>
+      )}
+    >
+      <Label>
+        {label}
+        <img src={question} alt="" />
+      </Label>
+    </OverlayTrigger>
+  )
+}
+
+function Docs({ isMobile }) {
+  return (
+    <ul>
+      <li>About</li>
```

Comment:
Let's add the links!

Reply:
Done already! But let's remove the `isMobile` prop, because it's not needed anymore. :)